src/components/Shared/BetForm.tsx

```diff
@@ -14,18 +14,27 @@ export interface GamePanelProps {
 
 const BetForm = (props: GamePanelProps) => {
     const [betResult, setBetResult] = useState<string>(props.bet?.gameResult || "");
+    const [betType, setBetType] = useState<string>(props.bet?.betType || "CLASSIC");
```

Comment:
Let's set the default value as "blank" (not "CLASSIC"), so we force the user (us) to choose an option.

```suggestion
    const [betType, setBetType] = useState<string>(props.bet?.betType || "");
```